app/src/main/java/org/p2p/wallet/history/model/rpc/RpcHistoryTransaction.kt

```diff
@@ -207,7 +207,9 @@ sealed class RpcHistoryTransaction(
             resources.getString(R.string.details_transfer_format, destination.cutMiddle(), symbol)
         }
 
-        fun getUsernameOrAddress(): String = counterPartyUsername ?: getAddress()
+        fun getUsernameOrAddress(): String = getUserName() ?: getAddress()
```

Comment:
getFormatttedUsernameOrAddress
as a user I would think that getUsernameOrAddress will return me exactly username